service/src/main/java/bio/terra/workspace/amalgam/tps/TpsApiController.java

```diff
@@ -54,4 +54,10 @@ public ResponseEntity<ApiTpsPaoGetResult> getPao(UUID objectId) {
     ApiTpsPaoGetResult result = tpsApiDispatch.getPao(bearerTokenFactory.from(request), objectId);
     return new ResponseEntity<>(result, HttpStatus.OK);
   }
+
+  @Override
+  public ResponseEntity<Void> clonePao(UUID objectId, UUID cloneObjectId) {
```

Comment:
Basic TPS question - Does TPS have access control, or can anyone call TPS?

For WSM, we check access control (is requester allowed to perform this action) in controller ([example](https://github.com/DataBiosphere/terra-workspace-manager/blob/main/service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java#L281)). I don't see any access control checks, so was curious.